### ISSUE TITLE
Uncaught TypeError: Cannot set property 'innerText' of undefined

### ISSUE THREAD

**Issue by rokyed** [camel-chased/editor-background#9]
[Enter steps to reproduce below:]

1. ...
2. ...

**Atom Version**: 1.0.3
**System**: Mac OS X 10.10.4
**Thrown From**: [editor-background](https://github.com/camel-chased/editor-background) package, v1.0.1

### Stack Trace

Uncaught TypeError: Cannot set property 'innerText' of undefined

```
At /Users/rokyed/.atom/packages/editor-background/lib/editor-background.coffee:957

TypeError: Cannot set property 'innerText' of undefined
  at Object.module.exports.EditorBackground.applyBackground (/Users/rokyed/.atom/packages/editor-background/lib/editor-background.coffee:957:9)
  at /Users/rokyed/.atom/packages/editor-background/lib/editor-background.coffee:200:33
  at /Applications/Atom.app/Contents/Resources/app.asar/src/config.js:557:16
  at /Applications/Atom.app/Contents/Resources/app.asar/src/config.js:574:20
  at Emitter.module.exports.Emitter.emit (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/event-kit/lib/emitter.js:82:11)
  at Config.module.exports.Config.emitChangeEvent (/Applications/Atom.app/Contents/Resources/app.asar/src/config.js:736:29)
  at Config.module.exports.Config.setRawValue (/Applications/Atom.app/Contents/Resources/app.asar/src/config.js:551:19)
  at Config.module.exports.Config.set (/Applications/Atom.app/Contents/Resources/app.asar/src/config.js:233:14)
  at SettingsPanel.module.exports.SettingsPanel.set (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/settings-view/lib/settings-panel.js:190:28)
  at HTMLSelectElement.<anonymous> (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/settings-view/lib/settings-panel.js:204:26)
  at HTMLSelectElement.jQuery.event.dispatch (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/jquery/dist/jquery.js:4435:9)
  at HTMLSelectElement.elemData.handle (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/jquery/dist/jquery.js:4121:28)

```

### Commands

```
 12x -3:42.9.0 core:backspace (atom-text-editor#tree-view-background.opacity.editor.mini.is-focused)
     -3:09.0 core:move-left (atom-text-editor#tree-view-background.opacity.editor.mini.is-focused)
 67x -3:07.6.0 core:backspace (atom-text-editor#tree-view-background.opacity.editor.mini.is-focused)
     -3:01.0 core:move-right (atom-text-editor#tree-view-background.opacity.editor.mini.is-focused)
     -3:00.9.0 core:backspace (atom-text-editor#tree-view-background.opacity.editor.mini.is-focused)
  2x -2:49 core:close (atom-text-editor#tree-view-background.opacity.editor.mini.is-focused)
     -2:23.7.0 settings-view:open (atom-pane.pane.active)
     -0:17.5.0 application:new-file (atom-workspace.workspace.scrollbars-visible-when-scrolling.theme-atom-material-syntax.theme-atom-material-ui.editor-background.editor-background)
     -0:06.4.0 core:close (atom-text-editor.editor.is-focused)
```

### Config

```json
{
  "core": {
    "themes": [
      "atom-material-ui",
      "atom-material-syntax"
    ],
    "projectHome": "/www/mainline"
  },
  "editor-background": {
    "box3d": {},
    "image": {
      "url": "http://media3.giphy.com/media/fJ1eCBjmMoEvu/giphy.gif",
      "style": "background-repeat: repeat;",
      "manualBackgroundSize": "400px 400px"
    },
    "other": {},
    "text": {},
    "video": {}
  }
}
```

### Installed Packages

```coffee
# User
atom-material-syntax, v0.2.1
atom-material-ui, v0.5.9
editor-background, v1.0.1
git-diff-details, v0.19.0
git-log, v0.4.1
minimap, v4.12.1
pdf-view, v0.23.0
pigments, v0.9.3
tabs-to-spaces, v0.11.0
tree-view-background, v0.3.2

# Dev
No dev packages
```

**Comment by camel-chased** [comment]
I think this is because of tree-view-background plugin, there can be a conflict

**Comment by theodorecackowski** [comment]
I don't have tree-view-background, and I was lead here,
but the editor-background plugin seems to still be working

**Comment by LukeXuan** [comment]
When I split a left pane, it lead me to here.

**Comment by camel-chased** [comment]
@LukeXuan in the latest version of the plugin? hmm
EB doesn't support panes yet but on my pc just only one panel is shadowed under the text, no error shows up on split... hmm
can you take a screenshot of this bug?

**Comment by LukeXuan** [comment]
I cannot reproduce it now. XD
I was just split a compiled js to left pane, and it reports this error.
And then there was two layers of background image.
Now it works simply that the second pane has no background. XD
I will notify you if I reproduce it again.

**Comment by dander401** [comment]
I just got this error. 
First time this has happened and I can't get to happen again, but heres what I did when it happened

1) Started atom
2) My last project was opened with the file I had.
3) I noticed the background not active so I toggled it through the cmd-shift-p menu.
4) got a message saying i had to open a file before it would start ( I was viewing a file )
5) opened the tree menu. EB activated ( image became the background) and the error was thrown

Hope this helps
